configs/openSUSE/pam-modules.toml

```diff
@@ -210,6 +210,14 @@ nodigests = [
     "*/security/pam_systemd.so",
 ]
 
+[[FileDigestGroup]]
+package = "systemd"
+type = "pam"
+note = "Module in systemd-experimental. This was not audited, if it's moved to a different binary rpm it needs to be audited first"
```

Comment:
It appears to have been audited in the meantime:
https://bugzilla.opensuse.org/show_bug.cgi?id=1185285

Reply:
We can change it to
`note = "Module in systemd-experimental."`

Reply:
I changed the "note" field and added the bugzilla ticket number!